src/strongbox/toc.clj

```diff
@@ -98,105 +98,139 @@
+        label (:title keyvals)
+        label (if (empty? label)
+                (do (warn "addon with no \"Title\" value found:" dirname)
+                    (when use-defaults
+                      no-label-label))
+                label)
+
+        wowi-source (when-let [x-wowi-id (-> keyvals :x-wowi-id utils/to-int)]
+                      {:source "wowinterface"
+                       :source-id x-wowi-id})
+
+        ;;curse-source (when-let [x-curse-id (-> keyvals :x-curse-project-id utils/to-int)]
+        ;;               {:source "curseforge"
+        ;;                :source-id x-curse-id})
+
+        ;;tukui-source (when-let [x-tukui-id (-> keyvals :x-tukui-projectid utils/to-int)]
+        ;;               {:source "tukui"
+        ;;                :source-id x-tukui-id})
+
+        github-source (when-let [x-github (-> keyvals :x-github)]
+                        {:source "github"
+                         :source-id (utils/github-url-to-source-id x-github)})
+
+        github-website-source (when-let [x-website (-> keyvals :x-website)]
+                                (when (and (not github-source)
+                                           (.startsWith x-website "https://github.com"))
+                                  {:source "github"
+                                   :source-id (utils/github-url-to-source-id x-website)}))
+
+        source-map-list (when-let [items (->> [wowi-source github-source github-website-source
+                                               ;;curse-source tukui-source 
+                                               ]
+                                              utils/items
+                                              utils/nilable)]
+                          {:source-map-list items})
+
+        ignore-flag (when (some->> keyvals :version (clojure.string/includes? "@project-version@"))
+                      (debug (format "ignoring '%s': 'Version' field in .toc file is unrendered" dirname))
+                      {:ignore? true})
+
+        ;; todo: warning when interface version not defined.
+
+        interface-version-list (vec
+                                (distinct
+                                 (into (some-> keyvals :interface parse-interface-value)
+                                       (some-> keyvals :#interface parse-interface-value))))
+        interface-version-list (if (empty? interface-version-list)
+                                 (if use-defaults
+                                   [constants/default-interface-version]
+                                   [])
+                                 interface-version-list)
+        interface-version (first interface-version-list)
+
+        ;; note: even after the `distinct` above, it's still possible for the derived game tracks to be duplicates
+        game-track-list (vec (distinct (mapv utils/interface-version-to-game-track interface-version-list)))
+        game-track (first game-track-list)
```

Comment:
this doesn't seem accurate to use anymore. can we remove usage of `:game-track` in favour of `:game-track-list` too?